api/src/router/routes/contact.ts

```diff
@@ -20,11 +20,13 @@
 
 import { asyncMiddleware } from '@bcgov/common-nodejs-utils';
 import express from 'express';
-import { createContact } from '../../controllers/contact';
+import { createContact, inviteToOrg } from '../../controllers/contact';
 
 const router = express.Router();
 
 // Contact
 router.post('/', asyncMiddleware(createContact));
+router.post('/inviteToOrg', asyncMiddleware(inviteToOrg));
```

Comment:
Note for me to look into later too, but what permissions are needed here, are we at risk of someone hitting the API endpoint and adding to our org without actually being allowed there?

Reply:
good point, did you mean from an external service? My assumption is that if a user is authenticated they can hit this endpoint yes?